tests/sentry/mediators/sentry_app_installations/test_installation_notifier.py

```diff
@@ -32,26 +42,90 @@ def test_task_enqueued(self, safe_urlopen):
         InstallationNotifier.run(
             install=self.install,
             user=self.user,
+            action='created',
         )
 
-        safe_urlopen.assert_called_once_with(
-            'https://example.com',
-            json={
-                'action': 'installation',
-                'installation': {
-                    'uuid': self.install.uuid,
+        data = faux(safe_urlopen).kwargs['data']
+
+        assert data == {
+            'action': 'created',
+            'installation': {
+                'uuid': self.install.uuid,
+            },
+            'data': {
+                'app': {
+                    'uuid': self.sentry_app.uuid,
+                    'slug': self.sentry_app.slug,
                 },
-                'data': {
-                    'app': {
-                        'uuid': self.sentry_app.uuid,
-                        'slug': self.sentry_app.slug,
-                    },
-                    'organization': {
-                        'slug': self.org.slug,
-                    },
-                    'uuid': self.install.uuid,
-                    'code': self.install.api_grant.code,
+                'organization': {
+                    'slug': self.org.slug,
                 },
+                'uuid': self.install.uuid,
+                'code': self.install.api_grant.code,
+            },
+            'actor': {
+                'id': self.user.id,
+                'name': self.user.name,
+                'type': 'user',
             },
-            timeout=5,
+        }
+
+        assert faux(safe_urlopen).kwarg_equals('headers', DictContaining(
+            'Content-Type',
+            'Request-ID',
+            'Sentry-Hook-Resource',
+            'Sentry-Hook-Timestamp',
+            'Sentry-Hook-Signature',
+        ))
+
+    @patch('sentry.mediators.sentry_app_installations.installation_notifier.safe_urlopen')
+    def test_uninstallation_enqueued(self, safe_urlopen):
+        InstallationNotifier.run(
+            install=self.install,
+            user=self.user,
+            action='deleted',
         )
+
+        data = faux(safe_urlopen).kwargs['data']
+
+        assert data == {
+            'action': 'deleted',
```

Comment:
I'm keeping this whole payload in the test for the 'deleted' case incase down the road we change it to be different from install, and so we (aka me) don't forget to test the whole payload and not just the action